integration/src/test/scala/org/knora/webapi/E2ESpec.scala

```diff
@@ -120,7 +120,7 @@ abstract class E2ESpec
   protected def singleAwaitingRequest(
     request: HttpRequest,
     timeout: Option[zio.Duration] = None,
-    printFailure: Boolean = false,
+    printFailure: Boolean = true,
```

Comment:
Just to make sure, this is desired change, not forgotten in cleanup?

Reply:
I am turning this on anyway any time I try to understand why the tests fails. I think it is also helpful on CI.